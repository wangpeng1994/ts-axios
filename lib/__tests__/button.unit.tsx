import renderer from 'react-test-renderer'; // 通过安装 jest for react-app 时，安装的虚拟渲染器
import React from 'react';
import Button from '../button';

describe('button', () => {
  it('是个 div', () => {
    const json =  renderer.create(<Button />).toJSON();
    expect(json).toMatchSnapshot();
  });
});