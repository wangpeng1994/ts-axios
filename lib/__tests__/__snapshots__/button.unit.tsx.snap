// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`button 是个 div 1`] = `
<div
  className="button"
  style={
    Object {
      "color": "red",
    }
  }
>
  点我啊
</div>
`;
